oma-pm/src/matches.rs

```diff
@@ -43,28 +43,40 @@ pub enum MatcherError {
     DpkgError(#[from] DpkgError),
 }
 
+/// Represent the type of package search engine.
 pub enum SearchEngine {
     Indicium(Box<dyn Fn(usize)>),
     Strsim,
     Text,
 }
 
+/// Represent the method of getting platform arch.
 pub enum GetArchMethod<'a> {
     SpecifySysroot(&'a Path),
     SpecifyArch(&'a str),
     DirectRoot,
 }
 
 #[derive(Builder)]
+
 pub struct PackagesMatcher<'a> {
+    /// The summary of all the apt operations in rust-apt
```

Comment:
Summary of all pending APT operations...

"apt" is only appropriate to represent the command, for the project/tool, use "APT."